src/extensions/spikeamplitudes/SpikeAmplitudesView/SpikeAmplitudesView.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react'
+import sizeMe, { SizeMeProps } from 'react-sizeme'
+import { SortingViewProps } from "../../extensionInterface"
+import Splitter from '../../timeseries/TimeWidgetNew/Splitter'
+import SelectUnitsWidget from './SelectUnitsWidget'
+import SpikeAmplitudesTimeWidget from './SpikeAmplitudesTimeWidget'
+
+const SpikeAmplitudesView: FunctionComponent<SortingViewProps & SizeMeProps> = ({recording, sorting, selection, selectionDispatch, size, hither}) => {
```

Comment:
Here's the new view that includes a unit select window (left) and the spike amplitudes widget (right)